ui/push-health/Job.jsx

```diff
@@ -82,9 +88,9 @@ Job.propTypes = {
     id: PropTypes.number.isRequired,
     result: PropTypes.string.isRequired,
     failure_classification_id: PropTypes.number.isRequired,
+    job_type_name: PropTypes.string.isRequired,
+    job_type_symbol: PropTypes.string.isRequired,
   }).isRequired,
-  jobName: PropTypes.string.isRequired,
```

Comment:
Not sure why I was passing in the `job` and then ALSO passing in a few fields from `job`.  Madness...